components/gallery/UnlockableTag.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <NeoTooltip
+    :active="!isOwner"
+    label="unlock-able content is revealed after purchase"
+    :append-to-body="true">
+    <div
+      class="unlockable-container is-flex border py-2 px-6 is-justify-content-space-between">
+      <div class="is-flex is-align-items-center">
+        <img class="mr-2" :src="unloackableIcon" alt="Unlockable Icon" />
+        <span class="has-text-grey is-size-7">Unlock-Able item</span>
+      </div>
+      <nuxt-link
+        v-if="isOwner"
+        :to="{
+          path: `/${urlPrefix}/gallery/${nft.id}`,
+          query: { unlockable: true },
+        }"
+        class="has-text-link">
+        {{ `Claim ${isMobile ? '' : 'Physical'} Drop` }}
+      </nuxt-link>
+
+      <span v-else>Owner Only</span>
+    </div>
+  </NeoTooltip>
+</template>
+
+<script lang="ts" setup>
+import unloackableDark from '@/assets/unlockable-dark.svg'
+import unloackable from '@/assets/unlockable.svg'
+import { NeoTooltip } from '@kodadot1/brick'
+import { isOwner as checkOwner } from '@/utils/account'
+import { NFT } from '@/components/rmrk/service/scheme'
+import { useWindowSize } from '@vueuse/core'
+
+const props = defineProps<{
+  nft: NFT | undefined
+}>()
+
+const { isDarkMode } = useTheme()
+const { accountId } = useAuth()
+const { urlPrefix } = usePrefix()
+const isMobile = computed(() => useWindowSize().width.value < 768)
```

Comment:
also different value

Reply:
??

Reply:
galley Item page changes layout at 930px
so that's the point where it make sense to move to unlock able tag up

but there is no need to cut short the text until mobile screen i.e 768 px